swig/acados.i

```diff
@@ -177,6 +177,7 @@ static std::string generate_vde_function(casadi::Function& model) {
     casadi::Function vde = casadi::Function(full_name, input, output);
     casadi::Dict opts;
     opts["with_header"] = casadi::GenericType(true);
+    opts["with_export"] = casadi::GenericType(false);
```

Comment:
Out of curiosity; why do you write explicit type-conversion to ` casadi::GenericType`?

Reply:
Good question, I was wondering the same. `true` should work as well, no?

Reply:
I remember getting an error at some point and just sticked to it.

Reply:
I merged it and fixed it locally.

Just `true` instead of `GenericType(true)` works for me.